webapp/channels/src/components/channel_name_form_field/channel_name_form_field.scss

```diff
@@ -0,0 +1,5 @@
+.channel-name-input-field {
+    height: 34px !important;
+    border: 0 !important;
+    border-radius: 0 !important;
+}
```

Comment:
To override the generic styling for this specific case.

Reply:
Actually, I'm seeing this was previously specified with `!important` before you moved it out into its own component. I'm still curious why it was needed to begin with, but since you're just moving it, I'm fine leaving it as-is

Reply:
yup, so keeping it as in.